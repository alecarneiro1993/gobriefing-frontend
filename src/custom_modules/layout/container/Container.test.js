import React from "react";
import { shallow } from "enzyme";

import { Container } from ".";

describe("Container", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<Container>Test</Container>);
    expect(wrapper).toMatchSnapshot();
  });
});
