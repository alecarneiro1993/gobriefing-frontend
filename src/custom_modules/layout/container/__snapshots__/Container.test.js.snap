// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders without crashing 1`] = `
<Styled(WithStyles(ForwardRef(Container)))>
  <Styled(WithStyles(ForwardRef(Grid)))
    container={true}
  >
    Test
  </Styled(WithStyles(ForwardRef(Grid)))>
</Styled(WithStyles(ForwardRef(Container)))>
`;
